index.html

```diff
@@ -777,7 +777,7 @@ <h3 property="schema:name"><span>Note</span>: Subject Verification</h3>
                   <div class="note" id="origin-considerations" inlist="" rel="schema:hasPart" resource="#origin-considerations">
                     <h3 property="schema:name"><span>Note</span>: Origin Considerations</h3>
                     <div datatype="rdf:HTML" property="schema:description">
-                      <p>Typical Web browsers use origin based security. While it has many limitations, it is however ubiquitous.</p>
+                      <p>Typical Web browsers use origin-based security. While it cannot be used to identify clients, it can, for instance, protect servers with MTLS-based authentication.</p>
```

Comment:
```suggestion
                      <p>Typical web browsers use origin-based security to isolate content and prevent malicious interactions. While not designed to identify clients, it can complement security mechanisms such as mTLS-based authentication.</p>
```